janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -358,15 +358,17 @@ public HBaseStoreManager(org.janusgraph.diskstorage.configuration.Configuration
 
     @Override
     public Deployment getDeployment() {
-        if (null == deployment) {
-            List<KeyRange> local;
-            try {
-                local = getLocalKeyPartition();
-                deployment = null != local && !local.isEmpty() ? Deployment.LOCAL : Deployment.REMOTE;
-            } catch (BackendException e) {
-                // propagating StorageException might be a better approach
-                throw new RuntimeException(e);
-            }
+        if (null != deployment) {
+            return deployment;
+        }
+
+        List<KeyRange> local;
+        try {
+            local = getLocalKeyPartition();
+            deployment = null != local && !local.isEmpty() ? Deployment.LOCAL : Deployment.REMOTE;
+        } catch (BackendException e) {
+            // propagating StorageException might be a better approach
```

Comment:
if the [interface of this class](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/diskstorage/common/DistributedStoreManager.java#L139) allows throwing on the getDeployment method, then StorageException would work, but its not in the definition, and it seems as if its driven by configuration. so, maybe a StorageException is not appropriate. delete comment?